bug: `ion-popover` is missing `side` input for Angular bindings

### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [X] v6.x

### Current Behavior

According to v6 [popover documentation](https://ionicframework.com/docs/api/popover#positioning), there should be a "side" attribute available for choosing which side to display the popover.

But I can't see this attribute available in the [current implementation](https://github.com/ionic-team/ionic-framework/blob/main/angular/src/directives/overlays/popover.ts) of the popover angular directive.

Am I completely missing the point or is something missing?

### Expected Behavior

The "side" attribute on the ion-popover element should allow me to choose on which side of the reference point the popover displays.

### Steps to Reproduce

```
<ion-popover trigger="add-item-button" dismiss-on-select side="top">    <= Adding "side" attribute has no effect
  <ng-template>
    <ion-content>
Some code
    </ion-content>
  </ng-template>
</ion-popover>
```


### Code Reproduction URL

_No response_

### Ionic Info

```
[ERROR] Error loading @angular-devkit/schematics package.json: Error: Cannot find module      
        '@angular-devkit/schematics/package'
        
        Require stack:
        - C:\Users\Guillaume Petit\AppData\Roaming\npm\node_modules\ionic\lib\project\index.js
        - C:\Users\Guillaume Petit\AppData\Roaming\npm\node_modules\ionic\lib\index.js        
        - C:\Users\Guillaume Petit\AppData\Roaming\npm\node_modules\ionic\index.js
        - C:\Users\Guillaume Petit\AppData\Roaming\npm\node_modules\ionic\bin\ionic

Ionic:

   Ionic CLI                     : 5.4.16 (C:\Users\Guillaume Petit\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 6.0.0
   @angular-devkit/build-angular : 13.1.2
   @angular-devkit/schematics    : not installed
   @angular/cli                  : 13.1.2
   @ionic/angular-toolkit        : 4.0.0

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v14.17.0 (D:\Development\Tools\nodejs\node.exe)
   npm    : 7.10.0
   OS     : Windows 10
```

### Additional Information

_No response_

--- a/angular/src/directives/overlays/popover.ts
+++ b/angular/src/directives/overlays/popover.ts
@@ -66,6 +66,7 @@ export declare interface IonPopover extends Components.IonPopover {
     'triggerAction',
     'reference',
     'size',
+    'side',
   ],
   methods: ['present', 'dismiss', 'onDidDismiss', 'onWillDismiss'],
 })
@@ -92,6 +93,7 @@ export declare interface IonPopover extends Components.IonPopover {
     'triggerAction',
     'reference',
     'size',
+    'side',
   ],
 })
 export class IonPopover {

